package-lock.json

```diff
@@ -186,6 +186,7 @@
         "style-loader": "2.0.0",
         "stylelint": "13.13.1",
         "stylelint-config-idiomatic-order": "8.1.0",
+        "stylelint-config-recommended": "5.0.0",
```

Comment:
It seems like stylelint might be mucking around with our ESLint config since I've seen it pop up before. I think it's fine to leave it here, but if it starts appearing and disappearing randomly in the future, we can look into getting rid of it. There was another library that used to do that before, so it's not the first time something like this happenedd.